### ISSUE TITLE
Update recent transaction's UI to handle transfer list (indexed-db migration 5)

### ISSUE THREAD

**Issue by sreejithkaimal** [hashgraph/hedera-wallet-ios#99]
<a href="https://github.com/enerestar"><img src="https://avatars3.githubusercontent.com/u/19986938?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [enerestar](https://github.com/enerestar)**
_Friday Mar 15, 2019 at 06:24 GMT_
_Originally opened as https://github.com/swirlds/clients-hedera/issues/301_

----

Current UI for recent transactions.

![Screen Shot 2019-03-18 at 5 36 13 PM](https://user-images.githubusercontent.com/19986938/54520818-8963e400-49a4-11e9-8cdd-13bab9dccf96.png)

With the implementation of a list of recipients for crypto transfer, how will users see recent transaction's page, since the list of recipients can range from 1 - n. 
1) Do we want to just render the total value of the transfer?  or
2) The whole list of users and amount tied to that crypto transfer?

@donaldthibeau @atulmahamuni

**Comment by sreejithkaimal** [comment]
<a href="https://github.com/donaldthibeau"><img src="https://avatars2.githubusercontent.com/u/42808726?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [donaldthibeau](https://github.com/donaldthibeau)**
_Monday Mar 18, 2019 at 12:43 GMT_

----

From a user perspective, I would likely want to know which publisher I paid, and the amount I paid them. Given the use case is for media publishing and consumption, I would therefore expect this list to show who was paid, and how much was paid to them. The alternative which would also be an expected experience would be to show the recipient, and the total amount I have paid that recipient over time.

**Comment by sreejithkaimal** [comment]
<a href="https://github.com/calvinchengx"><img src="https://avatars3.githubusercontent.com/u/95563?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [calvinchengx](https://github.com/calvinchengx)**
_Friday Mar 22, 2019 at 04:52 GMT_

----

The current local database (using indexed-db) managed by `hedera-browser-extension` has a single table (table named as `transactions`), with columns `transactionId`, `accountID`, `host`, `path`, `amount`, `created` and `receipt`.

We will now need to implement a second table named as `recipients`.  This new table will have a foreign key column that is linked to the `transactionId` column in the `transactions` table and additionally, will have a column `accountID` and `amount`.

The original table `transactions` will no longer need the `accountID` column.  The `amount` column is now the sum of all `amount` value in the `recipients` table, where `transactionId` is the foreign key.

In other words, a one-to-many relationship.

We need to implement a migration to create this new table and migrate old data existing in user's currently installed extension, to be consistent with this schema migration.

**Comment by sreejithkaimal** [comment]
<a href="https://github.com/calvinchengx"><img src="https://avatars3.githubusercontent.com/u/95563?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [calvinchengx](https://github.com/calvinchengx)**
_Friday Mar 22, 2019 at 04:56 GMT_

----

Retrieving the recipients (for a particular tx, there can be one or more recipients) for a particular tx will therefore use a query code of this form (pseudocode):-

```
db.recipients
  .where('transactionId').equals('0.0.1001@1549809381.3511') // can be replaced with other queries
  .then(recipients => {
      // Let's print the result:
      recipients.forEach (r => {
          console.log (`Recipient: ${r.accountID}`)
          console.log (`Amount: ${r.amount}`)
      });
})
```

**Comment by sreejithkaimal** [comment]
<a href="https://github.com/calvinchengx"><img src="https://avatars3.githubusercontent.com/u/95563?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [calvinchengx](https://github.com/calvinchengx)**
_Friday Mar 22, 2019 at 04:57 GMT_

----

In relation to our new recent transactions UI, we can therefore implement either a flat table, that repeats the same transaction with each recipient in that same transaction one or more times.

Or, we can render only the aggregate, and when user clicks on the aggregate list item, expand out to render each recipient and corresponding amount for each recipient.
